### ISSUE TITLE
[Install issue] - pg-native missing dependency after pg-promise installed

### ISSUE THREAD

**Issue by fluxquantum** [vitaly-t/pg-promise#786]
### Expected behavior
pg-promise lib runs after npm install.

### Actual behavior
pg-promise fails compilation:
```
./node_modules/pg-promise/node_modules/pg/lib/native/client.js
Module not found: Can't resolve 'pg-native' in '/Users/.../Documents/.../node_modules/pg-promise/node_modules/pg/lib/native'
```

### Steps to reproduce
Run npm install pg-promise@10.10.2 --dev

### Environment

* Version of pg-promise: 10.10.2
* OS type (Linux/Windows/Mac): Macos
* Version of Node.js: 10.12.0

**Comment by vitaly-t** [comment]
Looks like you installed old 10.9.2, and didn't really install the latest. Delete the library from `node_modules`, and properly install the library. The latest 10.10.2 doesn't have any problem.

**Comment by fluxquantum** [comment]
@vitaly-t Thanks for following up. I deleted pg-promise in the node_modules folder and I reinstalled it with the version unspecified to make sure it'd pull latest, but still seeing the error.

The package.json shows the correct version:
```
{
  "_from": "pg-promise@^10.10.2",
  "_id": "pg-promise@10.10.2",
  "_inBundle": false,
  "_integrity": "sha512-ezc5J......BEGR+tns...........N8p2m8nNrcrJ8uhM/g==",
  "_location": "/pg-promise",
  "_phantomChildren": {
    "buffer-writer": "2.0.0",
    "packet-reader": "1.0.0",
    "pg-types": "2.2.0",
    "pgpass": "1.0.4"
  },
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "pg-promise@^10.10.2",
    "name": "pg-promise",
    "escapedName": "pg-promise",
    "rawSpec": "^10.10.2",
    "saveSpec": null,
    "fetchSpec": "^10.10.2"
  },
......
```
Is there a dependency that I need to resolve as a pre-requisite?
I have pg-format and pg already as a dependency in my own package.json.

**Comment by vitaly-t** [comment]
Are you using [pgNative] initialization option?

[pgNative]:http://vitaly-t.github.io/pg-promise/module-pg-promise.html

**Comment by fluxquantum** [comment]
@vitaly-t I followed your wiki to initialize the pg promise db connections. I didn't explicitly set the flag.
https://github.com/vitaly-t/pg-promise/wiki/Connection-Syntax#configuration-object

Am I essentially using it if I did the following?

```
const pg_promise = require('pg-promise');

const pg_connection_config = {
    database: 'db.name',
    host: 'db.host',
    user: dbUs,
    password: dbPa,
    port: 5432,
    max: 15,
    idleTimeoutMillis: 30000
}

const pg_promise_db_connect = pg_promise(pg_connection_config);
```

**Comment by vitaly-t** [comment]
You usage example is invalid, see my own example I showed just earlier, and it is in documentation.

**Comment by fluxquantum** [comment]
Yea, I am not sure what is going on. I tried just putting **const pg_promise = require('pg-promise')();**
 in my react js file, and still getting the same error. 

I referenced https://github.com/vitaly-t/pg-promise/blob/31ed0034ca0b6fbda7551c7d6a94dcf567fbc937/jsdoc/README.md

I am also noticing this error show up as well in the console when I try to refresh the app.
```
index.js:11 Uncaught TypeError: Cannot read property 'split' of undefined
    at Object.<anonymous> (index.js:11)
    at Object../node_modules/pg-promise/lib/index.js (index.js:27)
    at __webpack_require__ (bootstrap:851)
    at fn (bootstrap:150)
```
With pgNative being a runtime flag, it is odd that it looks like it's failing at compile time.
